Modules/Sources/WordPressShared/Reachability/ReachabilityUtils.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+@objc
+public class ReachabilityUtils: NSObject {
+
+    @objc
+    public static func isInternetReachable() -> Bool {
+        (UIApplication.shared as? NetworkConnectionAvailabilityGetting)?.connectionAvailable ?? false
```

Comment:
Is this supposed to be using the app delegate instance?
```suggestion
        (UIApplication.shared.delegate as? NetworkConnectionAvailabilityGetting)?.connectionAvailable ?? false
```

Reply:
I feel like you could extract the `AppDelegate.configureReachability()` part to here too, so that you don't have to indirectly rely on the app delegate instance.